app/addons/documents/changes/actions.js

```diff
@@ -66,7 +61,7 @@ export default {
     var query = $.param(params);
     var db = app.utils.safeURLName(changesStore.getDatabaseName());
 
-    var endpoint = FauxtonAPI.urls('changes', 'server', db, '');
+    var endpoint = FauxtonAPI.urls('changes', 'server', db, '?' + query);
     currentRequest = $.getJSON(endpoint);
     currentRequest.then(_.bind(this.updateChanges, this));
```

Comment:
Yeah, `.then()` robs the `this` context so updateChanges can't reference `this.getLatestChanges()`.